fs/compress.py

```diff
@@ -76,7 +77,13 @@ def write_zip(src_fs,
                 )
             zip_info = zipfile.ZipInfo(zip_name, zip_time)
 
+            try:
```

Comment:
Information for external_attr was hard to find!

Reply:
I ended up reading the `zipfile.py` source indeed ^_^